### PR TITLE
Spring release notes

### PR DESCRIPTION
**Pull request by mrm9084**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,8 +39,10 @@ Module | Platform | Sample | Release Notes
...
+[azure-spring-cloud-azure-feature-management-web](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/appconfiguration/azure-spring-cloud-feature-management-web)<br/>[![Maven Central](https://img.shields.io/maven-central/v/com.azure.spring/azure-spring-cloud-feature-management-web.svg?color=blue)](https://search.maven.org/artifact/com.azure.spring/azure-spring-cloud-feature-management-web) | Spring Boot | [Sample](https://github.com/Azure-Samples/azure-spring-boot-samples/tree/main/appconfiguration/feature-management-web-sample) | [Release Notes](https://github.com/Azure/AppConfiguration/blob/main/releaseNotes/AzureSpringCloudFeatureManagement.md)
```
Comment (AlexandraKemperMS) [inline README.md:43]: This has an extra azure, it is supposed to be azure-spring-cloud-feature-management-web

File: README.md [hunk 2]
```diff
@@ -27,8 +27,10 @@ Module | Platform | Sample | Release Notes
...
+[spring-cloud-azure-appconfiguration-config-web](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/appconfiguration/spring-cloud-azure-appconfiguration-config-web)<br/>[![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/spring-cloud-azure-appconfiguration-config-web.svg?color=blue)](https://search.maven.org/artifact/com.microsoft.azure/spring-cloud-azure-appconfiguration-config-web) | Spring Boot | [Sample](https://github.com/Azure-Samples/azure-spring-boot-samples/tree/main/appconfiguration/azure-appconfiguration-sample) | [Release Notes](https://github.com/Azure/AppConfiguration/blob/main/releaseNotes/SpringCloudAzureAppConfigurationConfig.md)
```
Comment (zhenlan) [inline README.md:33]: I know we talked about keeping the "spring-cloud-*" stuff, but now I look at it and am afraid we may add more confusion than clarity. I suggest we keep the "azure-spring-*" stuff only. That's what we recommend at this point. Our documentation is updated to talk about that only. If there is another name changing in the future, we will update accordingly.

File: releaseNotes/AzureSpringCloudAppConfigurationConfig.md [hunk 3]
```diff
@@ -6,6 +6,11 @@
 
 [Source code web][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 2.1.0 - September 06, 2021
+
+* Added Health Monitor support with Spring Actuator.
+* Added new API which allows users to configure the behavior when a Key Vault reference which cannot be resolved with `KeyVaultSecretProvider`.
```
Comment (zhenlan) [inline releaseNotes/AzureSpringCloudAppConfigurationConfig.md:12]: Anywhere people can find more details of the new API, or should we include more details here?

File: releaseNotes/AzureSpringCloudAppConfigurationConfig.md [hunk 4]
```diff
@@ -6,6 +6,11 @@
 
 [Source code web][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 2.1.0 - September 06, 2021
+
+* Added Health Monitor support with Spring Actuator.
```
Comment (zhenlan) [inline releaseNotes/AzureSpringCloudAppConfigurationConfig.md:11]: Any github issue we can link?

File: README.md (hunk 2 above)
Comment (mrm9084) [reply README.md]: Removed, only have azure-spring now.

File: releaseNotes/AzureSpringCloudAppConfigurationConfig.md (hunk 4 above)
Comment (mrm9084) [reply releaseNotes/AzureSpringCloudAppConfigurationConfig.md]: Added a link.

File: releaseNotes/AzureSpringCloudAppConfigurationConfig.md (hunk 3 above)
Comment (mrm9084) [reply releaseNotes/AzureSpringCloudAppConfigurationConfig.md]: Added so API details.

File: releaseNotes/AzureSpringCloudAppConfigurationConfig.md [hunk 5]
```diff
@@ -6,6 +6,27 @@
...
+        @Override
+        public String getSecret(String uri) {
+
+            ...
+
+            return mySecret;
+        }
+    };
+}
+```
```
Comment (zhenlan) [inline releaseNotes/AzureSpringCloudAppConfigurationConfig.md:28]: NIT: can we add indentation, so the sample aligns with the text?

**Review by zhenlan — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (AlexandraKemperMS) [reply README.md]: I agree - a customer who is not as familiar with the process will just get confused, it all sounds similar

**Review by AlexandraKemperMS — APPROVED** [review]

**Pull request merged**